x-pack/metricbeat/module/mssql/_meta/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM microsoft/mssql-server-linux:2017-latest
```

Comment:
One additional thing I just spotted here: We normally defined the depenendcy up to the bug fix release to make sure CI always uses the exact same version as we had issues in the past. Not sure if we have here a more fine grained version or if this is already it.

Reply:
Actually, there's a `GA` version. I have already changed it